src/connector.js

```diff
@@ -703,16 +689,58 @@ function initEngine() {
 
 	// Listen to "Enter" key up event for search suggestions
 	if ( searchBoxElement ) {
+		searchBoxElement.onkeydown = ( e ) => {
+			// Enter
+			if ( e.keyCode === 13 && ( activeSuggestion != 0 && suggestionsElement && !suggestionsElement.hidden ) ) {
+				closeSuggestionsBox();
+				e.preventDefault();
+			}
+			// Escape or Tab
+			else if ( e.keyCode === 27 || e.keyCode === 9 ) {
+				closeSuggestionsBox();
+
+				if ( e.keyCode === 27 ) {
+					e.preventDefault();
+				}
+			}
+			// Arrow key up
+			else if ( e.keyCode === 38 ) {
+				if ( !( isFirefox && waitForkeyUp ) ){
+					waitForkeyUp = true;
+					searchBoxArrowKeyUp();
+					e.preventDefault();
+				}
+			}
+			// Arrow key down
+			else if ( e.keyCode === 40 ) {
+				if ( !( isFirefox && waitForkeyUp ) ){
+					waitForkeyUp = true;
+					searchBoxArrowKeyDown();
+				}
+			}
+		}
```

Comment:
Add semicolon for readability purposes (as flagged by our build script):
```suggestion
		};
```

Reply:
Done in latest commit